README.md

```diff
@@ -316,6 +319,27 @@ Please note that if you do not need to pass through Options if you will not be u
 String response = connection.get_job_status(partnerParameters.get(), null);
 ```
 
+##### get_hosted_web_session method
```

Comment:
As this is the first implementation of this feature I think we should take a moment to think about the naming . Is this what we want to call it? @camerondgray your thoughts?

Reply:
Good call to stop and be thoughtful about this. My instinct would be to call it `get_web_token` and update the rest of the comments to be consistent around that. I am definitely open to other suggestions though.

Reply:
`get_web_token` makes more sense.